conans/server/launcher.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python
+import argparse
+import os
+
+from conans import SERVER_CAPABILITIES
+from conans import __version__ as SERVER_VERSION, REVISIONS
+from conans.model.version import Version
+from conans.paths import conan_expand_user
+from conans.server.conf import MIN_CLIENT_COMPATIBLE_VERSION
+from conans.server.conf import get_server_store
+
+from conans.server.crypto.jwt.jwt_credentials_manager import JWTCredentialsManager
+from conans.server.crypto.jwt.jwt_updown_manager import JWTUpDownAuthManager
+from conans.server.migrate import migrate_and_get_server_config
+from conans.server.plugin_loader import load_authentication_plugin
+from conans.server.rest.server import ConanServer
+
+from conans.server.service.authorize import BasicAuthorizer, BasicAuthenticator
+
+
+class ServerLauncher(object):
```

Comment:
The rename of the file doesn't help to review :S

Reply:
The gunicorn launching of the server is documented and cannot be broken, so the module name cannot be changed. At the same time, crossed imports blocked the possibility of init the application with the migration flag so that change was mandatory in order to implement the fix, sorry.